src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -170,7 +163,8 @@
   [rules :- Rules
    request :- ring/Request
    name :- schema/Str]
-  (if-let [ { matched-rule :rule matches :matches } (some #(match? % request) rules)]
+  (if-let [{matched-rule :rule matches :matches}
+           (some #(match? % request) (sort-rules rules))]
```

Comment:
Yeah, we could do that.  I did the sorting here to ensure that rules would be checked in sorted order, in the case where something somehow was added or done to the rules after `transform-code` (and we wouldn't know if they were sorted correctly anymore).